suitcase/tiff_stack/tests/tests.py

```diff
@@ -24,3 +24,27 @@ def test_export(tmp_path, example_data):
         actual = tifffile.imread(str(filename))
         streamname = os.path.basename(filename).split('-')[0]
         assert_array_equal(actual, expected[streamname])
+
+
+def test_file_prefix_formatting(file_prefix_list, example_data, tmp_path):
+    ''' runs a test of the file_prefix formatting.
+
+    ..note::
+
+        Due to the `file_prefix_list` and `example_data` `pytest.fixture`'s
+        this will run multiple tests each with a range of file_prefixes,
+        detectors and event_types. see `suitcase.utils.conftest` for more info.
+
+    '''
+    collector = example_data()
+    file_prefix = file_prefix_list()
+    artifacts = export(collector, tmp_path, file_prefix=file_prefix)
+
+    for name, doc in collector:
+        if name == 'start':
+            templated_file_prefix = file_prefix.format(**doc).partition('-')[0]
```

Comment:
Maybe add a `break` there once the condition is satisfied?

Reply:
fixed